.github/workflows/infection.yaml

```
@@ -31,6 +31,6 @@ jobs:
         uses: "ramsey/composer-install@v1"
 
       - name: Run Infection
-        run: vendor/bin/roave-infection-static-analysis-plugin --threads=$(nproc)
+        run: vendor/bin/roave-infection-static-analysis-plugin --threads=$(nproc) --psalm-config=infection-psalm.xml.dist
```

```suggestion
        run: vendor/bin/infection --threads=$(nproc)
```

uz me ten psalm nebavi :D